_data/people.yaml

```diff
@@ -2654,6 +2654,21 @@ lauracarter:
   pronouns: she/her
   twitter: LauraC_rter
   website: https://lauracarter.github.io
+lalmehlisy:
```

Comment:
Could you move it between kswhitney and landimi2? Thank you :)

Reply:
Done! let me know if you need any other changes!